### ISSUE TITLE
Add more complexity on some country logomarks

### ISSUE THREAD

**Issue by caneco** [caneco/laravel-country-logomarks#8]
Hello,

I had this in the early stages of the logomarks but I've decided to removed it for the initial release.

**Would your like to have this _accents_ on some selected logomarks?**

## Before

![image](https://user-images.githubusercontent.com/502041/98594338-6d752a80-22cc-11eb-9352-c5e1f71b19e2.png)

## After

![image](https://user-images.githubusercontent.com/502041/98594354-736b0b80-22cc-11eb-9fa4-f013390bc858.png)

**Comment by ianrodrigues** [comment]
Nah

**Comment by abrardev99** [comment]
Yes. Depend on the flag I think. If the flag has a texture logo should have too. I guess 😸

**Comment by caneco** [comment]
@abrardev99 yup… depends on the flag.

Maybe I will add as an optional asset(?) 🤔

**Comment by caneco** [comment]
Here's another case that with a bit of complexity it helps connecting the logomark with the actual flag.

## Normal
![image](https://user-images.githubusercontent.com/502041/98714226-8b9c6280-2380-11eb-8ed3-b0c71b2be558.png)

## With complexity 
![image](https://user-images.githubusercontent.com/502041/98714249-93f49d80-2380-11eb-99ce-72bbe51bacc2.png)

**Comment by abrardev99** [comment]
Wow, I'm more liking the second one. Normal is also feels nice and minimalist. Thank You @caneco

**Comment by Wulfheart** [comment]
Coming from #3.
The pakistani logo looks nice with more complexity. However the stars in the US variant make it look uneasy. 
Also consider that it takes more time to insert a country's enblem in the logo. So I strongly recommend keeping it simple but therefore adding more flags.

**Comment by caneco** [comment]
Another scenario; in this case for the Canada (CA) logomark

## Simple
![image](https://user-images.githubusercontent.com/502041/99003577-32762f80-2536-11eb-8da7-41c6c7ce1b63.png)


## With complexity
![image](https://user-images.githubusercontent.com/502041/99003605-3b670100-2536-11eb-955f-5971c852ed86.png)

**Comment by Wulfheart** [comment]
I find this one like the Pakistani appealing. Guess it were the stars which made it look uneasy for me.

**Comment by brada1703** [comment]
I like logomark with complexity for Canada 👍  ❤️
